library/simulation/racecar_core_sim.py

```diff
@@ -20,6 +20,7 @@
 import drive_sim
 import lidar_sim
 import physics_sim
+import drone_sim
```

Comment:
You need to change `__VERSION = 1` to 2 since you are updating the communication protocol.